### ISSUE TITLE
setup options not working

### ISSUE THREAD

**Issue by firefoxNX** [scottie1984/swagger-ui-express#19]
In the generated html I am getting

```
var customOptions = this["undefined"] = {
  "operationsSorter": "alpha",
  "docExpansion": "none"
};
```

I am using below customization

```//serve swagger-ui
    const showExplorer = false;
    const suiOptions = {
        operationsSorter: function (a, b) {
            let order = {
                'get': '0',
                'post': '1',
                'put': '2',
                'patch': '3',
                'delete': '4'
            };
            return order[a.get('method')].localeCompare(order[b.get('method')]);
        },
        docExpansion: 'none'
    };
    const customCss = '.swagger-ui .topbar { display: none }';
    app.use(constants.SERVICE_BASE_URI + '/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument, showExplorer, suiOptions, customCss));```

**Comment by firefoxNX** [comment]
If I change [index.js line # 23](https://github.com/scottie1984/swagger-ui-express/blob/master/index.js#L23) to ```var indexHTML = htmlWithSwaggerReplaced.replace('<% customOptions %>', stringify(options, 'options'))``` and [index.js line # 45](https://github.com/scottie1984/swagger-ui-express/blob/master/index.js#L45) to ```return json + ';';``` then it works as expected.

**Comment by scottie1984** [comment]
Thanks - should be fixed in 2.0.1 which I just published
